wiki/osu!_wiki/Contribution_guide/GitHub_Web_Interface/en.md

```diff
@@ -0,0 +1,88 @@
+# GitHub web interface
```

Comment:
this article was removed on purpose in https://github.com/ppy/osu-wiki/pull/6437, right @TicClick?

Reply:
yes. didn't notice it's back -- please remove it from the PR well